packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -1,3 +1,5 @@
+import { PromiseType } from 'react-use/lib/util';
+
 import TrezorConnect, { Device } from 'trezor-connect';
```

Comment:
```suggestion
import TrezorConnect, { Device } from 'trezor-connect';

import type { PromiseType } from 'react-use/lib/util';
```

We try to use type only imports and exports. (not for a long time, so you will definitely find imports which violate it)

Reply:
Fixed in [`5eec1822`](https://github.com/trezor/trezor-suite/commit/5eec1822f8e6504b0a2dc0ff955f37d141c0f921).